### PR TITLE
MEAN and PostgreSQL oneclick guides [1k]

### PR DESCRIPTION
**Pull request by Rajakavitha1**
Created one click guides for:
- PosgreSQL
- MEAN

### CODE REVIEW

**Comment by nmelehan** [comment]
Deploy preview for *nostalgic-ptolemy-b01ab8* ready!

Built with commit 196ff220a9f86ecd0561d56b134f0d050f98f694

https://deploy-preview-3208--nostalgic-ptolemy-b01ab8.netlify.com

**Comment by nmelehan** [comment]
Deploy preview for *nostalgic-ptolemy-b01ab8* ready!

Built with commit 3d488f42a9b996d251a95b3dad404dfea73d87bd

https://deploy-preview-3208--nostalgic-ptolemy-b01ab8.netlify.com

File: docs/platform/one-click/deploy-postresql-with-one-click-apps/index.md [hunk 1]
```diff
@@ -0,0 +1,155 @@
...
+
+    This is the PostgreSQL client shell, in which you can issue SQL commands. To see a list of available commands, use the `\h` command. You may find more information on a specific command by adding it after `\h`.
+
+### Create Tables
+
+This section contains examples which create a test database with an employee's first and last name, assigning each a unique key. When creating your own tables, you may specify as many parameters (columns) as you need and name them appropriately. Run the commands in this section from the PostgreSQL client shell, that you opened to create `mytestdb` database.
+
+1.  Create a table called "employees" in your test database:
+
+        CREATE TABLE employees (employee_id int, first_name varchar, last_name varchar);
```
Comment (andystevensname) [inline docs/platform/one-click/deploy-postresql-with-one-click-apps/index.md]: Should employee_id be SERIAL PRIMARY KEY? Just to show how to auto increment? Up to you to decide if this is necessary.

File: docs/platform/one-click/deploy-mean-with-one-click-apps/index.md [hunk 2]
```diff
@@ -0,0 +1,72 @@
+---
+author:
+  name: Linode Community
+  email: docs@linode.com
+description: 'Deploy the MEAN stack on Linode with One-Click Apps.'
```
Comment (andystevensname) [inline docs/platform/one-click/deploy-mean-with-one-click-apps/index.md]: Also need an og_description.

Comment (Rajakavitha1) [reply docs/platform/one-click/deploy-mean-with-one-click-apps/index.md]: fixed

File: docs/platform/one-click/deploy-mean-with-one-click-apps/index.md [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+| **Root Password** | The primary administrative password for your Linode instance. This password must be provided when you log in to your Linode via SSH. It must be at least 6 characters long and contain characters from two of the following categories: lowercase and uppercase case letters, numbers, and punctuation characters. Your root password can be used to perform any action on your server, so make it long, complex, and unique. *Required*. |
+
+
+### Getting Started After Deployment
+
+After your MEAN One-click App has finished installing, you can:
+
+- [Connect to your Linode via SSH](/docs/getting-started/#connect-to-your-linode-via-ssh). You will need your Linode's root password to proceed.
+
+- Consult the following guides to learn more about working with the various components of the MERN stack:
```
[the remark this answers is not in the document]
Comment (Rajakavitha1) [reply docs/platform/one-click/deploy-mean-with-one-click-apps/index.md]: fixed!

File: docs/platform/one-click/deploy-postresql-with-one-click-apps/index.md [hunk 4]
```diff
@@ -0,0 +1,155 @@
...
+        psql (12.2 (Debian 12.2-2.pgdg90+1))
+        Type "help" for help.
+
+        mytestdb=#
+
+    This is the PostgreSQL client shell, in which you can issue SQL commands. To see a list of available commands, use the `\h` command. You may find more information on a specific command by adding it after `\h`.
+
+### Create Tables
+
+This section contains examples which create a test database with an employee's first and last name, assigning each a unique key. When creating your own tables, you may specify as many parameters (columns) as you need and name them appropriately. Run the commands in this section from the PostgreSQL client shell, that you opened to create `mytestdb` database.
```
[the remark this answers is not in the document]
Comment (Rajakavitha1) [reply docs/platform/one-click/deploy-postresql-with-one-click-apps/index.md]: fixed.

File: docs/platform/one-click/deploy-postresql-with-one-click-apps/index.md (hunk 1 above)
Comment (Rajakavitha1) [reply docs/platform/one-click/deploy-postresql-with-one-click-apps/index.md]: fixed

**Pull request merged**